docs/docs/05-reusable-components.md

```diff
@@ -294,7 +294,18 @@ ReactDOM.render(<HelloMessage name="Sebastian" />, mountNode);
 ```
 
 This simplified component API is intended for components that are pure functions of their props. These components must not retain internal state, do not have backing instances, and do not have the component lifecycle methods. They are pure functional transforms of their input, with zero boilerplate.
-However, you may still specify `.propTypes` and `.defaultProps` by setting them as properties on the function, just as you would set them on an ES6 class.
+However, you may still specify `.propTypes` and `.defaultProps` by setting them as properties on the function, just as you would set them on an ES6 class:
```

Comment:
@TanaseHagi can you add a new line here, per [@zpao's request](https://github.com/facebook/react/pull/7458#discussion_r74297117)? I think we can merge this after that change.